app/services/application/ApplicationService.scala

```diff
@@ -114,9 +115,10 @@ trait ApplicationService extends EventSink with CurrentSchemeStatusHelper {
       calculateCurrentSchemeStatus(current, SchemeEvaluationResult(withdrawRequest.schemeId, EvaluationResults.Withdrawn.toString) :: Nil)
     }
 
-    def maybeProgressToFSAC(schemeStatus: Seq[SchemeEvaluationResult]) = {
+    def maybeProgressToFSAC(schemeStatus: Seq[SchemeEvaluationResult], latestProgressStatus: Option[ProgressStatus]) = {
       val greenSchemes = schemeStatus.collect { case s if s.result == Green.toString => s.schemeId }.toSet
-      val shouldProgressToFSAC = greenSchemes subsetOf schemesRepo.nonSiftableSchemeIds.toSet
+      val shouldProgressToFSAC = !latestProgressStatus.contains(ProgressStatuses.SIFT_READY) &&
+        (greenSchemes subsetOf schemesRepo.nonSiftableSchemeIds.toSet)
```

Comment:
I think this `(greenSchemes subsetOf schemesRepo.nonSiftableSchemeIds.toSet)` should be extracted into another variable for readability.